tests/conftest.py

```diff
@@ -53,6 +53,48 @@ def pytest_logger_config(logger_config):
     logger_config.set_formatter_class(OCSLogFormatter)
 
 
+@pytest.fixture()
```

Comment:
shouldn't this be a session scoped fixture? Or is the intention to use this fixture in specific tests?

Reply:
This should be called just for a specific subset of tests. I am adding this for now because tests related to capacity alerts are flaky for configurations that don't meet minimal requirements. I am will add another PR that will improve logging and debugging of these tests (and other) if they have sufficient configuration.

Reply:
Thanks for the explanation @fbalak